app/socket/notify.js

```diff
@@ -0,0 +1,45 @@
+/* eslint no-console: 0 */
+import { SOCKET_URL } from 'constants/index';
+import io from 'socket.io-client';
+
+const link = `${SOCKET_URL}notify`;
+
+const notifyHub = (function () {
```

Comment:
Given that this is client-side where they'll only have on socket on the `/notify` namespace, do you really need to wrap it in a function here? You're transpiling ES6 modules so you could simply keep the socket reference in the module without being exported. It would lead to cleaner code without the wrapped function.

Alternatively you could consider a class to wrap the socket.